yaml/pois.yaml

```diff
@@ -1131,5 +1131,4 @@ filters:
       tourism: attraction
     # already clamped below polygon area min
     min_zoom: 17
-    output:
-      kind: {col: tourism, tier: 6}
+    output: { kind: attraction, tier: 6}
```

Comment:
While this is correct, I suggest separating onto individual lines like:

```
output:
  kind: attraction
  tier: 6
```

For example:

https://github.com/paulmach/vector-datasource/blob/628a0aa905f3146694644f75cd65f2c247f71fd5/yaml/pois.yaml#L159-L165